workers/clustering_worker/trained_crf_model/directory.md

```diff
@@ -0,0 +1 @@
+for models
```

Comment:
Why is this file here?

Reply:
Its a GitHub "trick".  If a directory is empty, it will not push it to the repository. However, we need this directory because that's where the models are generated. A better solution would be a `mkdir` command in the worker, however the risk there is installation on a file system where the user has restricted rights for making directories.  We have 2 secure installations where this is the case.

Reply:
Can have a .gitkeep too

Reply:
@aksh555 : Can I just put that in the root, or is it better in the subdirectory? We do *not* want to keep models from instance to instance. Unless my understanding of the models is wrong? I think they are tuned to local data.

Reply:
@aksh555 : I put a .gitkeep into the local worker directories that are empty when we begin. I think that fixes the issue.

Reply:
`.gitkeep` is fine, thanks for clarifying